controllers/storageclass/storageclass_controller_test.go

```diff
@@ -216,6 +221,86 @@ var _ = Describe("StorageClassWatcher", func() {
 			}, timeout, interval).Should(BeTrue())
 		})
 
+		It("should get FlashSystemCluster by StorageClass successfully", func() {
```

Comment:
please add more test cases - like creating StorageClass with two matches FlashSystemClusters and different pool name.